pkgs/development/python-modules/unearth/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pytestCheckHook
+, packaging
+, requests
+, cached-property
+, pdm-pep517
+}:
+
+buildPythonPackage rec {
+  pname = "unearth";
+  version = "0.6.1";
+  format = "pyproject";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-S3v719NKEWc9gN+uf6u/khwTmqx4OD+wyhapDTtTpm4=";
+  };
+
+  nativeBuildInputs = [
+    pdm-pep517
+  ];
+
+  propagatedBuildInputs = [
+    packaging
+    requests
+  ] ++ lib.optionals (pythonOlder "3.8") [
+    cached-property
+  ];
+
```

Comment:
Needs to either disable `doCheck` or configure explicit `checkInputs` (like `pytestCheckHook`) or an explicit `checkPhase` for running tests.

Reply:
how tf did i miss that ...

Reply:
hm it has a [test-dependency](https://pypi.org/project/requests-wsgi-adapter/) that is not yet in nixpkgs but got it's last release in 2019.
should i also add that to nixpkgs, or add it in here?

any opinion?

Reply:
i added the package to `pythonPackages`